utils/config.go

```diff
@@ -65,5 +66,9 @@ func WaitForLogMessage(t *testing.T, snap, expectedLog string, since time.Time)
 		}
 	}
 
-	t.Fatalf("Time out: reached max %d retries.", maxRetry)
+	t.Logf("Time out: reached max %d retries.", maxRetry)
+	stdout, _, _ := Exec(t,
+		fmt.Sprintf("sudo journalctl --lines=10 --no-pager --unit=snap.\"%s\".otbr-agent --priority=notice", snap))
```

Comment:
This will query the logs of a particular app (otbr-agent) and makes assumptions on the priority level and required number of lines to have sufficient debugging information.

The function could make use of a refactored SnapLogs function, rather than writing a new command.